src/install.js

```diff
@@ -69,6 +70,8 @@ async function main() {
     : install remoteresources3decrypt at a specific version (Default 'latest')
 --mtp, --mustachetemplate=''
     : install mustachetemplate at a specific version (Default 'latest')
+--iw, --impersonationwebhook=''
+    : install impersonation webhook at a specific version (Default 'latest'). When remote resource controller and/or mustache template controller are installed, this webhook will be installed even if this flag is not set
```

Comment:
similar to how we do `--rd-cluster-metadata64`, i think we want another option after `--impersonationwebhook` to allow a user to specify base64 encoded ca certs. This would give them a way to override the default self signed certs without having to manually go update those resources on cluster. (i assume we probably need 3 options, one for each `ca.crt`, `tls.crt`, and `tls.key`, or maybe we can group them into a single json object and have the whole thing be base64 encoded like we do for `--rd-cluster-metadata64`

Reply:
agreed. I can add that.